app/uk/gov/hmrc/apprenticeshiplevy/connectors/DesConnector.scala

```diff
@@ -51,7 +52,7 @@ trait DesUtils {
 trait EmployerDetailsEndpoint extends Timer {
   des: DesConnector =>
 
-  def designatoryDetails(empref: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[DesignatoryDetails] = {
+  def designatoryDetails(empref: String)(implicit hc: HeaderCarrier): Future[DesignatoryDetails] = {
```

Comment:
Are any of the http calls in this file DES. If they are you will need to pass headers explicitly to them

Reply:
We have modified the app to explicitly send the headers. Also have introduced WireMock in DesConnector unit tests to allow us to verify that the headers were sent.

Reply:
Could you add a correlation id as well. It's easy enough just `"CorrelationId" -> UUID.randomUuid()` or something